src/components/KYCWall/BaseKYCWall.tsx

```diff
@@ -242,6 +243,7 @@ function KYCWall({
                     selectPaymentMethod(item);
                 }}
                 shouldShowPersonalBankAccountOption={shouldShowPersonalBankAccountOption}
+                // eslint-disable-next-line react/jsx-no-comment-textnodes
```

Comment:
Why was it added now? Doesn't seem connected to the PR 😅

Reply:
I removed it in the last commit, backstory is that I run a script to add these comments and it caused this